MAUI/StepProgressBar/migration.md

```diff
@@ -0,0 +1,425 @@
+</tr> 
+<tr>
+<td>{{'[StatusChangedEventArgs ]()' | markdownify }}</td>
+<td>{{'[StepStatusChangedEventArgs]()' | markdownify }}</td>
+<td>Represents a class which is used to hold the status changed event arguments.</td>
+</tr>
+</table>
+
+## Properties 
+
+### SfStepProgressBar
+
+The following code example, explains how to initialize the properties of the `Xamarin SfStepProgressBar` and `.NET MAUI SfStepProgressBar` class.
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+...
+
+SfStepProgressBar sfStepProgressBar = new SfStepProgressBar ();
+this.Content = sfStepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfStepProgressBar stepProgressBar = new SfStepProgressBar();
+this.Content = stepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>{{'[CompletedStepStyle]()'| markdownify }}</td>
+<td>{{'[CompletedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the completed step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[InProgressStepStyle]()'| markdownify }}</td>
+<td>{{'[InProgressStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the in progress step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[NotStartedStepStyle]()'| markdownify }}</td>
+<td>{{'[NotStartedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the not started step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[Orientation]()'| markdownify }}</td>
+<td>{{'[Orientation]()'| markdownify }}</td>
+<td>Gets or sets the orientation.</td>
+</tr>
+<tr>
+<td>{{'[ProgressAnimationDuration]()'| markdownify }}</td>
+<td>{{'[ProgressAnimationDuration]()'| markdownify }}</td>
+<td>Gets or sets the progress animation duration.</td>
+</tr>
+<tr>
+<td>{{'[TitleAlignment]()'| markdownify }}</td>
+<td>{{'[LabelPosition]()'| markdownify }}</td>
+<td> Gets or sets the label position or tile alignment.</td>
+</tr>
+<tr>
+<td>{{'[TitleSpace]()'| markdownify }}</td>
+<td>{{'[LabelSpacing]()'| markdownify }}</td>
+<td>Gets or sets the label spacing.</td>
+</tr>
+<tr>
+<td>{{'[LayoutOption]()'| markdownify }}</td>
+<td>Nil</td>
+<td>Not Supported.</td>
```

Comment:
{{'[LayoutOption]()'\| markdownify }} | Nil | Not Supported.
-- | -- | --

Checked alternative solution available?"

Reply:
Checked the LayoutOption API and we dont have alternative API in MAUI